config/environments/production.rb

```diff
@@ -119,5 +119,15 @@
                  port: Settings.redis.port, drive: :hiredis }
   }
 
+  config.middleware.insert_before 0, 'Rack::Cors', logger: (-> { Rails.logger }) do
+    allow do
+      origins /[a-z0-9]+\.sumofus\.org/Z/i
```

Comment:
What do you think about moving this config to an ENV var? I think it'd make sense if we want this project to be usable by other organizations.